tasks/kafka-cfg.yml

```diff
@@ -50,7 +50,7 @@
     - kafka-install
 
 - name: "Link alternate logs directory and touch output files"
-  shell: "( test ! -e /usr/local/kafka/logs || rm -rf /usr/local/kafka/logs ) && ln -s /var/log/kafka /usr/local/kafka/logs && touch /var/log/kafka/state-change.log /var/log/kafka/kafkaServer.out && chown kafka:kafka /var/log/kafka/state-change.log /var/log/kafka/kafkaServer.out"
+  shell: "( test ! -e /usr/local/kafka/logs || rm -rf /usr/local/kafka/logs ) && ln -s {{ kafka_log_dir }} /usr/local/kafka/logs && touch {{ kafka_log_dir }}/state-change.log {{ kafka_log_dir }}/kafkaServer.out && chown {{ kafka_user }}:{{ kafka_group }} {{ kafka_log_dir }}/state-change.log {{ kafka_log_dir }}/kafkaServer.out"
```

Comment:
Not specific to this PR, but this line looks pretty bad, IMO. What's wrong with `ls -sf`? Why not use Ansible's built-in `file` module? Why are there two different responsibilities (linking log dir and touching files) mashed together?

I've looked at the rest of this repo and it seems to be of mixed quality. Stuff like this (`ansible-kafka/tasks/kafka-install.yml`): 

```
- set_fact: use_systemd=True
  when: init.stdout == 'systemd'
  tags:
    - kafka-install

- set_fact: use_systemd=False
  when: init.stdout != 'systemd'
  tags:
    - kafka-install
```

could be, IMHO, just:

```
- set_fact: use_systemd="{{init.stdout == 'systemd'}}"
  tags:
    - kafka-install
```

And of questionable intent (`ansible-kafka/tasks/system.yml`):

```
  lineinfile: dest=/etc/profile.d/Z99-kafka.sh regexp="^export PATH=.*$" line='export PATH="${PATH}:/usr/local/lib/kafka/bin"' state=present create=yes
```